presto-raptor/src/test/java/com/facebook/presto/raptor/metadata/TestDatabaseShardManager.java

```diff
@@ -747,4 +781,17 @@ private static Node createTestingNode()
     {
         return new PrestoNode(UUID.randomUUID().toString(), URI.create("http://test"), NodeVersion.UNKNOWN, false);
     }
+
+    private int numberOfColumns(long tableId)
+            throws SQLException
+    {
+        int num = 0;
+        try (ResultSet rs = dummyHandle.getConnection().getMetaData()
+                .getColumns(null, null, shardIndexTable(tableId).toUpperCase(), null)) {
```

Comment:
I think this is wrong since it doesn't filter based on catalog/schema. Should probably use `getCatalog()` / `getSchema()` from `Connection`